cdap-examples/HelloWorld/src/test/java/co/cask/cdap/examples/helloworld/HelloWorldTest.java

```diff
@@ -57,7 +58,8 @@ public void test() throws TimeoutException, InterruptedException, IOException {
 
     try {
       // Wait for the last Flowlet processing 5 events, or at most 5 seconds
-      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics("HelloWorld", "WhoFlow", "saver");
+      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics(Constants.DEFAULT_NAMESPACE, "HelloWorld", "WhoFlow",
```

Comment:
is not cool. We need to hide it, until we expose namespace functionality to user